From 172b40ea7b0995a18e751b80bfec195733f287e8 Mon Sep 17 00:00:00 2001
From: Rasmus Thomsen <oss@cogitri.dev>
Date: Sun, 30 Aug 2020 12:40:35 +0200
Subject: [PATCH] fix(corecollector): fix setting CONFIG_PATH etc. via meson

Previously it could happen that we accidentally included the globals.d
in source instead of the configured copy, so we would keep @CONFIG_PATH@
in the actual binary
---
 source/corecollector/{globals.d => globals.d.in} | 0
 source/corecollector/meson.build                 | 2 +-
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename source/corecollector/{globals.d => globals.d.in} (100%)

diff --git a/source/corecollector/globals.d b/source/corecollector/globals.d.in
similarity index 100%
rename from source/corecollector/globals.d
rename to source/corecollector/globals.d.in
diff --git a/source/corecollector/meson.build b/source/corecollector/meson.build
index 2a1a97b..bd7e671 100644
--- a/source/corecollector/meson.build
+++ b/source/corecollector/meson.build
@@ -1,5 +1,5 @@
 globals_d = configure_file(
-    input: 'globals.d',
+    input: 'globals.d.in',
     output: 'globals.d',
     configuration: conf,
 )
-- 
2.28.0

